purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -556,18 +556,20 @@ internal class PurchasesOrchestrator(
     fun getCustomerInfo(
```

Comment:
Hmm just noticed this one is only called internally. Might be worth making this private?
```suggestion
    private fun getCustomerInfo(
```